sa/model.go

```diff
@@ -1355,6 +1355,6 @@ func newPBFromIdentifierModels(ids []identifierModel) (*sapb.Identifiers, error)
 type pausedModel struct {
 	identifierModel
 	RegistrationID int64      `db:"registrationID"`
-	PausedAt       *time.Time `db:"pausedAt"`
+	PausedAt       time.Time  `db:"pausedAt"`
```

Comment:
The comment needs updating now that PausedAt can't be `nil`. While you're at it, let's describe `unpausedAt`'s state as `nil` vs `non-nil`.